FusionMCPBridge/handlers/design/features.py

```
@@ -0,0 +1,183 @@
+        Response dictionary
+    """
+    # Import here to avoid circular imports
+    from ... import FusionMCPBridge
+    
+    points = data.get('points', [[0, 0]])
+    width = float(data.get('width', 1.0))
+    faceindex = int(data.get('faceindex', 0))
+    distance = data.get('depth', None)
+    if distance is not None:
+        distance = float(distance)
+    through = bool(data.get('through', False))
```

Ruff (0.14.10)

Local variable `through` is assigned to but never used

Remove assignment to unused variable `through`

(F841)